components/brave_wallet/browser/internal/orchard_block_scanner.h

```diff
@@ -43,9 +50,13 @@ class OrchardBlockScanner {
   // Scans blocks to find incoming notes related to fvk
   // Also checks whether existing notes were spent.
   virtual base::expected<Result, OrchardBlockScanner::ErrorCode> ScanBlocks(
-      std::vector<OrchardNote> known_notes,
+      OrchardTreeState tree_state,
       std::vector<zcash::mojom::CompactBlockPtr> blocks);
 
+  static Result CreateResultForTesting(
```

Comment:
Where is this used? Seems like it belongs in test utils in any case

Reply:
Are there unit tests for this class? Shouldn't they need changes?

Reply:
We have orchard_block_scanner_unittest which uses orchard_block_scanner inside

Reply:
Testing a class through another class is not a unit test.